onnxruntime/python/tools/quantization/quant_utils.py

```diff
@@ -71,7 +71,9 @@ def from_string(v):
 class QuantType(Enum):
     QInt8 = 0
     QUInt8 = 1
-    QFLOAT8E4M3FN = 2
+    QInt16 = 2
+    QUInt16 = 3
+    QFLOAT8E4M3FN = 4
```

Comment:
>QFLOAT8E4M3FN [](start = 4, length = 13)

Will it cause compatibility issue? We should avoid change existing value. Just add new type incrementally.

Reply:
Updated.
It won't cause a compatibility issue if the enum's integer value is not saved/used, but I agree that I should have just added the new type at the end. It is safer.